components/form/form-helper.js

```diff
@@ -6,9 +6,12 @@ export const isCustomFormElement = (node) => isCustomElement(node) && !!node.for
 
 export const tryGetLabelText = (ele) => {
 	if (ele.labels && ele.labels.length > 0) {
-		const labelText = ele.labels[0].textContent.trim();
-		if (labelText) {
-			return labelText;
+		const textNode = [...ele.labels[0].childNodes].find(node => node.nodeType === Node.TEXT_NODE);
+		if (textNode) {
+			const labelText = textNode.wholeText.trim();
```

Comment:
`wholeText` doesn't appear to exist [until Safari 13](https://caniuse.com/#feat=mdn-api_text_wholetext)... we're still supporting 10 unfortunately.

Reply:
Ouch, I saw IE11 on MDN and thought I'd be good to go, I'll swap it out for doing it manually.

Reply:
Cool or you could feature detect it and fall back to something else, 🤷

Reply:
I just went with `filter` + `reduce`, its not quite as fancy but should be more than sufficient given.